src/bench/bench.cpp

```diff
@@ -41,18 +42,44 @@ void GenerateTemplateResults(const std::vector<ankerl::nanobench::Result>& bench
 
 } // namespace
 
-benchmark::BenchRunner::BenchmarkMap& benchmark::BenchRunner::benchmarks()
+namespace benchmark {
+
+std::map<std::string, PriorityLevel> map_priority_level = {
```

Comment:
Since the previous force push, we now can't pass `-priority-level=all` anymore, I don't think that's intentional? And if it was, I don't think I agree with that `-priority-level=low,medium,high` is cumbersome and will silently be incomplete when adding new levels.

Reply:
> Since the previous force push, we now can't pass -priority-level=all anymore, I don't think that's intentional?

Yes, to avoid tags clashing problems. e.g. when you call `PriorityToString` with any value, the returned string was the actual priority and, as it matches with the rest of the tags, the "all" tag.

But.. will re-add the "all" shortcut differently now.

And.. you know, I actually forgot to go further here, should had removed the `PriorityLevel::ALL` and replace it by a plain 0xff.
We shouldn't allow benchmarks to be set under the `PriorityLevel::ALL` tag as you did above (the `BENCHMARK(AddrManAddThenGood, benchmark::PriorityLevel::ALL)`).

> And if it was, I don't think I agree with that -priority-level=low,medium,high is cumbersome and will silently be incomplete when adding new levels.

With the latest change (0xff inclusion), all levels will be automatically added there.